bitgouel-api/src/main/kotlin/team/msg/domain/auth/service/AuthServiceImpl.kt

```diff
@@ -109,6 +113,31 @@ class AuthServiceImpl(
         teacherRepository.save(teacher)
     }
 
+    /**
+     * 뽀짝샘 회원가입을 처리하는 비지니스 로직입니다.
+     * @param 뽀짝샘 회원가입을 처리하기 위한 request dto 입니다.
+     */
+    @Transactional(rollbackFor = [Exception::class])
+    override fun bbozzakSignUp(request: BbozzakSignUpRequest) {
+        val user = createUser(
+            request.email,
+            request.name,
+            request.phoneNumber,
+            request.password,
+            Authority.ROLE_TEACHER
```

Comment:
```suggestion
            Authority.ROLE_BBOZZAK
```
뽀짝샘 회원가입인데 권한을 취동샘으로 부여하네요. 수정 부탁드립니다!